### DISCUSSION TITLE
k3s binairy wont start after changes in /etc/systemd/system/k3s.service

### DISCUSSION THREAD

**Discussion by dekstiertje** [k3s-io/k3s#7563] [Q&A]
I wanted to add oicd configuration in /etc/systemd/system/k3s.service to add OIDC authentication but when i wanted to do: sudo systemctl daemon-reload and sudo systemctl restart k3s.service the services wont restart. 

if i do k3s server i get:  starting kubernetes: preparing server: failed to normalize token; must be in format K10<CA-HASH>::<USERNAME>:<PASSWORD> or <PASSWORD>

if i do: sudo cat /var/lib/rancher/k3s/server/token, the file is empty.

Can someone help me? :)

**Comment by brandond** [comment]
You should be able to delete the file and it will recreate it, although the fact that it is empty suggests that something else is going on. Did you manually configure the token in your CLI flags or config file? Can you share the contents of your k3s systemd unit, as you currently have it? I suspect that you've got something wrong in there.
